### PR TITLE
[DOC-REVIEW] getting-started-tutorial: edit the model-serving doc (Step 6), demo overviews, and view function doc

### PR DESCRIPTION
**Pull request by Sharon-iguazio**
Doc review for PR #274, #279 & PR #293.

### CODE REVIEW

**Comment by Sharon-iguazio** [comment]
Note that in the "Test the Model from the Dashboard" section, I changed the reference to concatenating `sklearn server` to the project name to concatenating `-model-server`, as this is what I see in the output for the subsequent code cell (`getting-started-iris-iguazio-model-server` for project "getting-started-iris-iguazio".

File: getting-started-tutorial/getting-started-tutorial.ipynb [hunk 1]
```diff
@@ -2880,14 +2870,9 @@
...
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Set the trained model and deploy the serving function"
+    "### Deploy the Serving Function\n",
+    "\n",
+    "Set the trained model and deploy the serving function."
```
Comment (gilad-shaham) [inline getting-started-tutorial/getting-started-tutorial.ipynb:2875]: Does this work with the TOC extension?

Comment (Sharon-iguazio) [reply getting-started-tutorial/getting-started-tutorial.ipynb:2875]: Yes. That's also what we do in the other NBs that I edited. This is a screen shot from the updated extension TOC::
image
The line that you highlighted in your comment is in the same cell as the "Deploy the Serving Model Function" in Step 6, and as you can see, it appears in the TOC, as do all other headings that also have text in the same cell. The important thing is to start the cell with the heading and not with an anchor, as we previously did.

Comment (gilad-shaham) [reply getting-started-tutorial/getting-started-tutorial.ipynb:2875]: okay, just verifying

**Review by gilad-shaham — APPROVED** [review]

**Pull request merged**